app/views/catalog/facet.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="facet_pagination top">
+  <%= render :partial=>'facet_pagination' %>
+</div>
+
+<div class="modal-header">
+  <button type="button" class="ajax-modal-close close" data-dismiss="modal" aria-hidden="true">×</button>
+
+  <h3 class="modal-title"><%= facet_field_label(@facet.key) %></h3>
+  <%= render partial: 'facet_index_navigation' if @facet.index_range && @display_facet.index? %>
+
+</div>
+<div class="modal-body">
+  <div class="facet_extended_list">
+    <%= render_facet_limit(@display_facet, layout: false) %>
+  </div>
+</div>
+
+<div class="modal-footer">
+
+  <div class="facet_pagination bottom">
+    <div class="col-sm-4">
+      <input type="text" id="filterInput" placeholder="Filter..." class="form-control form-control-sm" />
+    </div>
+    <%= render 'facet_pagination' %>
+  </div>
+</div>
+
+<script>
+$(document).ready(function() {
```

Comment:
Can we move this code into the asset pipeline (e.g. `app/assets/javascript/....`)? 

Can you use `Blacklight.onLoad` rather than `$(document).ready` (accounts for turbolinks events)

Reply:
Yes.

Reply:
would it work if you did `$(document).on('keyup', '#filterInput', function() {} )`?

Reply:
Yes, that worked. Fixed.